ansible_collections/arista/avd/roles/eos_designs/doc/node-types-v4.0.md

```diff
@@ -47,6 +47,7 @@ node_type_keys:
     mpls_lsr: < true | false | default -> false >
 
     # Optional | Override ip_addressing templates
+    # Path is relative to playbook
```

Comment:
This is not completely correct. They can also be pointing to the builtin templates inside the collection. But since we will soon remove those in favor of python, we can leave this.

I suggest to add a section below this code block to describe the full background similar to https://avd.sh/en/stable/roles/eos_cli_config_gen/index.html#extensibility-with-custom-templates
Then refer to that section in this comment. Because it is not only relative to playbook, it must also be _below_ the playbook dir.

Reply:
Maybe let's reference the ansible documentation as well: https://docs.ansible.com/ansible/latest/user_guide/playbook_pathing.html#resolving-local-relative-paths ?